Makefile

```diff
@@ -639,7 +638,6 @@ SPEED_OPTIMISED_SRC := $(SPEED_OPTIMISED_SRC) \
             drivers/rx_nrf24l01.c \
             drivers/rx_spi.c \
             drivers/rx_xn297.c \
-            drivers/pwm_output.c \
```

Comment:
Yeap.

1. Identify "Pure GYROPID loop" functions
2. Put them in different file and give it speed optimized (or the other way around), or
3. Use function attribute for optimization.

Everybody seems to have their own preference...